api/Services/DBService.cs

```diff
@@ -270,6 +287,20 @@ public async Task<List<Staff>> GetAllStaff()
 
             return queryDefinition;
         }
+        private QueryDefinition CreateNewGameQueryDefinition(
+            string containerName,
+            Game newEntry)
+        {
+            var queryString = $"INSERT INTO {containerName} (id, title, romanizedTitle, releases," +
```

Comment:
Once we get a few of these in here and can start to see some patterns I think we might want to refactor a little bit just so it's a little cleaner. For now this looks great.

Reply:
👍